packages/next-tinacms-github/README.md

```diff
@@ -108,6 +109,47 @@ export const getStaticProps: GetStaticProps = async function({
 }
 ```
 
+#### `getGithubFile`
+
+In some cases you'll need to load multiple files from Github. For this reason the underlying `getGithubFile` function is exposed.
+
+```ts
+import {
+  getGithubFile,
+  parseJson,
+  parseMarkdown,
+} from 'next-tinacms-github'
+
+export const getStaticProps: GetStaticProps = async function({
+  preview,
+  previewData,
+  ...ctx
+}) {
+  const githubOptions = {
```

Comment:
is there any reason why we need 
```ts
githubOptions = {
//..
}
const homeFile = await getGithubFile({
    ...githubOptions,
    fileRelativePath: "content/index.md
    parse: parseMarkdown
  })
```
is there any way we could like this easier and have the same param as getGithubPreviewProps so we could have
```js
const homeFile = await getGithubFile({
    ...previewData,
    fileRelativePath: "content/index.md
    parse: parseMarkdown
  })
```